apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -384,4 +391,12 @@ protected static int getNumParallelJobs(CommandLine cmdLine) {
     }
     return ret;
   }
+
+  protected static int getTaskTimeOut(CommandLine cmdLine) {
+    int ret = DEFAULT_TASKS_WAIT_TIMEOUT_HOURS;
+    if (cmdLine.hasOption("taskTimeOutInHours")) {
+      ret = Integer.parseInt(cmdLine.getOptionValue("taskTimeOutInHours"));
+    }
+    return ret;
```

Comment:
```suggestion
    return NumberUtils.toInt(cmdLine.getOptionValue("taskTimeOutInHours"), DEFAULT_TASKS_WAIT_HOURS);
```
Can just do it in place

Reply:
Ack. Updated this method alone and leaving other methods as such